backend/capellacollab/settings/modelsources/t4c/repositories/routes.py

```diff
@@ -120,7 +120,7 @@ def delete_t4c_repository(
         interface.delete_repository(instance, repository.name)
     except (fastapi.HTTPException, requests.RequestException) as e:
         log.error("Repository deletion failed partially", exc_info=True)
-        response.status_code = status.HTTP_207_MULTI_STATUS
+        response.status_code = status.HTTP_202_ACCEPTED
```

Comment:
The reason for multi-status is that it failed partially. So it's somewhere between 2xx and 4xx.

Accepted means: 
"The HyperText Transfer Protocol (HTTP) 202 Accepted response status code indicates that the request has been accepted for processing, but the processing has not been completed"

In our case, the request is completed. According to Mozilla, MULTI_STATUS should only be used as part of the WebDAV protocol, so `HTTP_422_UNPROCESSABLE_ENTITY` is maybe an option.

Reply:
Then 422 is probably the most appropriate.